druid_setup/environment

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# Java tuning
+DRUID_XMX=1g
+DRUID_XMS=1g
+DRUID_MAXNEWSIZE=250m
+DRUID_NEWSIZE=250m
+DRUID_MAXDIRECTMEMORYSIZE=6172m
+
+druid_emitter_logging_logLevel=debug
+
+# Extensions packaged by Druid.
+# Extensions provided by Zenysis
+druid_extensions_loadList=["druid-lookups-cached-global", "postgresql-metadata-storage", "druid-stats", "druid-datasketches", "/var/lib/zen/extensions/druid-aggregatable-first-last", "/var/lib/zen/extensions/druid-arbitrary-granularity", "/var/lib/zen/extensions/druid-nested-json-parser", "/var/lib/zen/extensions/druid-tuple-sketch-expansion"]
+
+druid_zk_service_host=zookeeper
+
+druid_metadata_storage_host=
+druid_metadata_storage_type=postgresql
+druid_metadata_storage_connector_connectURI=jdbc:postgresql://postgres:5432/druid
+druid_metadata_storage_connector_user=druid
+druid_metadata_storage_connector_password=FoolishPassword
+
+druid_coordinator_balancer_strategy=cachingCost
+
+druid_indexer_runner_javaOptsArray=["-server", "-Xmx1g", "-Xms1g", "-XX:MaxDirectMemorySize=3g", "-Duser.timezone=UTC", "-Dfile.encoding=UTF-8", "-Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager"]
+druid_indexer_fork_property_druid_processing_buffer_sizeBytes=256MiB
+
+druid_storage_type=local
+druid_storage_storageDirectory=/opt/shared/segments
+druid_indexer_logs_type=file
+druid_indexer_logs_directory=/opt/shared/indexing-logs
+
+druid_processing_numThreads=2
+druid_processing_numMergeBuffers=2
+
```

Comment:
Other assorted settings:

`druid_segmentCache_locations` is where historical puts the local file cache for active datasources and is a required property. It should be put in a volume (the actual path doesn't matter), but only historical needs access to it

```
druid_javascript_enabled=true
druid_coordinator_asOverlord_enabled=true
druid_coordinator_asOverlord_overlordService=druid/overlord
druid_indexer_runner_type=remote
druid_indexer_storage_type=metadata
druid_segmentCache_locations=[{"path": "/data/output/druid", "maxSize": 300000000000, "freeSpacePercent": 5}]
```